generators/app/index.js

```diff
@@ -9,27 +9,35 @@ var semver = require('semver');
 var versions = require('./dependency-versions.js');
 var yosay = require('yosay');
 var _ = require('lodash');
+var genConfig = null;
```

Comment:
Why are you using a global variable here rather than a variable inside the scope of our module/class (the Function that extends ```yeoman.Base```)?